src/Settings/Component.svelte

```diff
@@ -54,34 +50,80 @@
     <div class="font-nunito bg-primary absolute z-20 top-header h-page w-screen">
         <div class="w-5/6 m-auto flex flex-col gap-5 items-center text-secondary text-settings">
             <SettingsOption class="w-full max-w-xl flex flex-col items-baseline">
-                <div
-                    class="w-full flex justify-around items-stretch"
-                    on:click="{() => (isModeExpanded = !isModeExpanded)}"
-                >
-                    <span>{TEXT['Game input']}</span>
+                <RadioSetting
```

Comment:
Do we need `SettingsOption` and `RadioSetting` separated? I guess we could make `RadioSetting` use `SettingsOption` internally